fe/fe-core/src/main/java/com/starrocks/external/hive/HiveMetastoreApiConverter.java

```diff
@@ -185,7 +199,7 @@ public static Map<String, String> toHudiProperties(Table metastoreTable,
                                                        Schema tableSchema) {
         Map<String, String> hudiProperties = Maps.newHashMap();
 
-        String hudiBasePath = metastoreTable.getSd().getLocation();
+        String hudiBasePath = ObjectStorageUtils.formatObjectStoragePath(metastoreTable.getSd().getLocation());
```

Comment:
we use origin scheme as cache key. and it will modify in `RemoteFileIO.getRemoteFiles`. we may use different pool size to pull remote files according file scheme later